1-exercises/A-this/exercise-2.js

```diff
@@ -11,7 +11,9 @@ let person = {
     name: "Maggie",
     age: 26,
     currentAddress: "London",
-    changeAddress: /* todo */
+    changeAddress: function (newAddress) {
+        return person.newAddress;
+    }
```

Comment:
good , you can write this.currentAddress = newAddress to return.

Reply:
Thank you Nagehan!

Reply:
Thank you Nagehan!